src/lib/support/CHIPArgParser.cpp

```diff
@@ -43,6 +43,7 @@
 #include <stdarg.h>
 #include <stdint.h>
 #include <string.h>
+#include <strings.h>
```

Comment:
I think this shouldn't be needed once you switched to gnu++14

Reply:
It was indeed not needed, however purely from a `man strcasecmp` it seems to be technically correct.